depends/config.site.in

```diff
@@ -47,6 +47,10 @@ if test -z $enable_zmq && test -n "@no_zmq@"; then
   enable_zmq=no
 fi
 
+if test -n $enable_bip70 && test -n "@enable_bip70@"; then
+  enable_bip70=yes
```

Comment:
I don't think this is correct, since we've never made it such that bip70 is disabled by default. Instead it should be disable_bip70=no etc